filebeat/config/config.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Defaults for config variables which are not set
 const (
-	DefaultRegistryFile                      = ".filebeat"
+	DefaultRegistryFile                      = "registry"
```

Comment:
The reason I used `registry` is that on DEB/RPM installations the file name used is `/usr/share/filebeat/registry`. By using registry we can keep BC for those cases at least.

In addition, JSON is an implementation detail which we might change in the future. Also, adding the extension could be inviting people to edit the file, which I guess it's the last thing we want. 

I see the three +1, but is there a strong reason to add the `.json`?